druid-shell/src/window.rs

```diff
@@ -65,10 +65,18 @@ impl IdleHandle {
     where
         F: FnOnce(&dyn Any) + Send + 'static,
     {
-        self.0.add_idle(callback)
+        self.0.add_idle_callback(callback)
```

Comment:
I wonder if we even need to keep the closure based API?

We could even get fancy and have a single method that has the signature, `schedule_idle(&self, idle: impl Into<IdleKind>)` but i think that's too fancy for now.

I think what makes the most sense is that we merge this as-is, and we can worry about `add_idle` at some later point.

Reply:
Agreed. Let review it in future after we collect more data point of usage of these idle api.